### PR TITLE
styling

### PR DESCRIPTION
**Pull request by knehrybecki**
check :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,305 @@
+<html lang="en">
+
```
Comment (jpudysz) [inline index.html]: remove enter

File: index.html [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/b20687152d.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style/style.css">
+    <title>Website</title>
+</head>
+
```
Comment (jpudysz) [inline index.html]: remove enter

File: index.html [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+<body>
+    <main>
+        <header class="header">
+            <div class="header__logo">
+                <img src="image/logo.jpg" alt="" class="logo" />
```
Comment (jpudysz) [inline index.html]: I guess it should be a styled div, not image. This is simple gray box

File: index.html [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+                                    <i class="fas fa-shopping-cart"></i>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <div class="banner">
+            <img src="image/banner.jpg" alt="" class="banner__img" />
```
Comment (jpudysz) [inline index.html]: same here, use` border-radius` and `background-color`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+                    </a>
+                    <a href="">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </div>
+            </footer>
+    </main>
+    <script src="script.js"></script>
+</body>
+
```
Comment (jpudysz) [inline index.html]: remove enter

File: index.html [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+</html>
```
Comment (jpudysz) [inline index.html]: this sign means that you need to add one enter at the end of the file

File: script.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const menulist= document.querySelector(".menu__list");
+const buttonshow= document.querySelector(".fas.fa-bars");
+const buttonhidden= document.querySelector(".fas.fa-times")
+
+
```
Comment (jpudysz) [inline script.js]: remove one enter

File: script.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+const menulist= document.querySelector(".menu__list");
```
Comment (jpudysz) [inline script.js]: add spaces between `=` eg. `const menuList = document...`

File: script.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+const buttonshow= document.querySelector(".fas.fa-bars");
+const buttonhidden= document.querySelector(".fas.fa-times")
```
Comment (jpudysz) [inline script.js]: use camelCase eg. `buttonShow`

File: script.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
+const menulist= document.querySelector(".menu__list");
+const buttonshow= document.querySelector(".fas.fa-bars");
```
Comment (jpudysz) [inline script.js]: use singlequote: `'.das.fa-bars'`

File: script.js [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+const buttonhidden= document.querySelector(".fas.fa-times")
+
+
+buttonshow.addEventListener("click",function(){
+    menulist.style.display= "block";
```
Comment (jpudysz) [inline script.js]: don't use `;`

File: script.js [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+    buttonhidden.style.display= "block";
+})
+buttonhidden.addEventListener("click",function(){
+    menulist.style.display= "none";
+    buttonhidden.style.display= "none";
+})
+
+
+///i wrote this code for the first time, please on understand :)
```
Comment (jpudysz) [inline script.js]: enter to remove this sign

File: style/_base.sass [hunk 13]
```diff
@@ -0,0 +1,10 @@
+*
+    margin: 0
+    padding: 0
+    box-sizing: border-box
+
+body
+
+    background-color: $body-color
+    color: $text-color
+    line-height: 1.5em
```
Comment (jpudysz) [inline style/_base.sass]: aa

File: style/_mixin.sass [hunk 14]
```diff
@@ -0,0 +1,6 @@
+@mixin displayflex($flex-flow,$align-items,$justify-content)
+
+    display: flex
+    justify-content: $justify-content
+    align-items: $align-items
+    flex-flow: $flex-flow
```
Comment (jpudysz) [inline style/_mixin.sass]: aa

File: style/_resetcss.sass [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+    text-decoration-line: none
+    color: inherit
+
+li 
+    list-style: none
+
+input[type="search"]::-webkit-search-cancel-button,
+input[type="search"]::search-decoration 
+    -webkit-appearance: none
+    appearance: none
```
Comment (jpudysz) [inline style/_resetcss.sass]: aa

File: style/_resetcss.sass [hunk 16]
```diff
@@ -0,0 +1,19 @@
+button
+    border: none
```
Comment (jpudysz) [inline style/_resetcss.sass]: you can use reset.css

File: style/_virables.sass [hunk 17]
```diff
@@ -0,0 +1,14 @@
+//colors
+
+$body-color: #fafafa
+$header-color: #f6f6f6
+$text-color: #a1a1a1
+$h2-text: #737373
+$search-color: #e2e2e2
+
+
```
Comment (jpudysz) [inline style/_virables.sass]: remove unnecessary enters

File: style/_virables.sass [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+
+$phone: "(max-width: 480px)"
+$tablet: "(min-width: 481px) and (max-width: 839px)"
+$desktop: "(min-width: 840px) and (max-width: 1279px)"
+$large-desktop: "(min-width: 1280px)"
```
Comment (jpudysz) [inline style/_virables.sass]: enter

File: style/header/_action.sass [hunk 19]
```diff
@@ -0,0 +1,5 @@
+.action
+    &__item
+        @include displayflex(row wrap,center,center)
+        font-size: 22px
+        gap: 15px
```
Comment (jpudysz) [inline style/header/_action.sass]: aa

File: style/header/_header.sass [hunk 20]
```diff
@@ -0,0 +1,18 @@
...
+        width: 100%
+
+@media #{$desktop},#{$large-desktop}
+
+    .header
+        @include displayflex(row nowrap,center,space-around)
+        padding: 20px 10px 15px 10px
+
+        &__block
+            @include displayflex(row wrap,center,space-around)
```
Comment (jpudysz) [inline style/header/_header.sass]: aa

File: style/header/_logo.sass [hunk 21]
```diff
@@ -0,0 +1,13 @@
...
+    border-radius: 20px
+    margin: 30px 
+
+@media #{$desktop},#{$large-desktop}
+
+    .logo
+        width: 100px
+        height: 30px
+        margin: 0
+        margin-left: 20px
```
Comment (jpudysz) [inline style/header/_logo.sass]: aa

File: style/header/_menu.sass [hunk 22]
```diff
@@ -0,0 +1,59 @@
...
+        &__link
+            font-size: 15px
+
+@media #{$large-desktop}
+
+        .menu
+
+            &__list
+                gap: 40px
+                padding-left: 20px
```
Comment (jpudysz) [inline style/header/_menu.sass]: aa

File: style/header/_search.sass [hunk 23]
```diff
@@ -0,0 +1,23 @@
...
+        color: $text-color
+        outline: none
+
+    &__button
+       transform: translate(-137px, 2px)
+
+@media #{$desktop} 
+    
+    .search
+        margin-left: 0px
```
Comment (jpudysz) [inline style/header/_search.sass]: aa

File: style/main/_banner.sass [hunk 24]
```diff
@@ -0,0 +1,3 @@
+.banner__img
+    width: 100vw
+    height: 90vh
```
Comment (jpudysz) [inline style/main/_banner.sass]: aa

File: style/style.sass [hunk 25]
```diff
@@ -0,0 +1,15 @@
...
+@import "base"
+@import "resetcss"
+
+@import "header/header"
+@import "header/logo"
+@import "header/menu"
+@import "header/search"
+@import "header/action"
+
+@import "main/banner"
```
Comment (jpudysz) [inline style/style.sass]: aa

**Review by jpudysz — CHANGES_REQUESTED** [review]

**Review by jpudysz — APPROVED** [review]

**Comment by knehrybecki** [comment]
ok

**Pull request merged**